qemu/tests/numa_stress.py

```diff
@@ -87,6 +87,8 @@ def run(test, params, env):
     most_used_node, memory_used = max_mem_map_node(host_numa_node, qemu_pid)
 
     for test_round in range(test_count):
+        if os.path.isfile(memory_file):
```

Comment:
@CongLi   I tested in @yanan-fu 's host with smaller memory for guest, and it passed.   It turns out that it is affected by the host free memory.  I will add some code before the test. Thanks for your review.